blog/tidy-finance-dummy-data/index.qmd

```diff
@@ -0,0 +1,422 @@
+---
+title: "Dummy Data for Tidy Finance Readers without Access to WRDS"
+author:
+  - name: Christoph Scheuch
+    url: https://christophscheuch.github.io/
+    affiliations:
+      - name: wikifolio Financial Technologies AG
+date: "2023-09-20"
+description: R code to generate dummy data that can be used to run the code chunks in Tidy Finance with R or Python
+image: thumbnail.png
+image-alt: An image of a stylized company building on a screen in the middle of a room. The room has a futuristic setting, with a backdrop of a digital, grid-like landscape symbolizing the internet. The color palette should be a combination of cool blues and warm yellow. Created with DALL-E 2.
+categories: 
+  - Data
+---
+
+Since we published our book [Tidy Finance with R](../../r/index.qmd), we have received feedback from readers who don't have access to WRDS that they cannot run the code we provide. To alleviate their constraints, we decided to create a dummy database that contains all tables and corresponding columns such that all code chunks in our book can be executed with this dummy database. The resulting database can be found through [this link](https://1drv.ms/u/s!ArgZyNRLSLEXgspompdvaUcBIzfekw?e=3R1L0t) (around 50 MB). Just download the database and put it into you data folder (I already renamed it to `tidy_finance.sqlite`).
```

Comment:
Typo correction: "Just download the database and put it into **your** data folder (I already renamed it to `tidy_finance.sqlite`)."